string location_request
---
string frame_id
float32 pose_x
float32 pose_y

